dir.props

```diff
@@ -452,6 +453,12 @@
         <NuGetTargetMoniker>.NETCoreApp,Version=v1.0</NuGetTargetMoniker>
       </PropertyGroup>
     </When>
+    <When Condition="'$(TargetGroup)'=='netcoreapp1.1'">
```

Comment:
we are in dev/api branch which is why I asked this :smile: , but that's ok, we can do it in a different PR.